### DISCUSSION TITLE
Demande d'information

### DISCUSSION THREAD

**Discussion by Paul-Che** [FredM67/PVRouter-3-phase#9] [Q&A] [answered]
Bonjour Fred, 

Je me présente, Paul, j'ai lu avec beaucoup d'intérêt vos messages et votre travail. Je conviens que ce n'est pas le lieu pour parler de cela mais j'aimerais avoir votre avis d'expert. Je rénove ma maison et je suis en passe d'acheter un BEC qui sera compatible avec une future installation PV avec routeur du surplus. J'ai compris qu'il est préférable d'avoir un BEC avec résistante blindée, quel modèle me recommandé vous ? 
De manière général, j'ai un peu de connaissance mais je suis loin d'être expert et je ne souhaite pas m'aventurer sur du DIY arduino et autre. Je préfère passer par un installateur pour avoir les assurances etc. 
Si vous deviez installer une prod PV en autoconsommation avec renvoi du surplus dans un BEC, quel serait votre installation idéal en terme de matériel et de rapport qualité/prix sachant que j'ai un tableau électrique en triphasé déjà. 

Je vous remercie pour votre temps et si vous ne souhaitez pas répondre car ce n'est pas le lieu je m'en excuse je n'ai pas trouvé d'alternative. Vous pourrez supprimer mon message. 

Merci Fred, 

Paul

**Comment by FredM67** [comment]
Bonjour Paul,
Je ne recommande pas spécialement un ballon à résistance blindée. Autant que je sache, la plupart dispose de ballon avec résistance stéatite (dont moi !). Un ballon avec thermostat mécanique sera plus simple à câbler avec un routeur qu'un ballon avec carte ACI. Néanmoins, il existe une solution pour ce genre de ballon SANS modifier la carte, cf. l'article dédié dans la Wiki.
Personnellement, je vous recommande une installation PV avec sytème Enphase (micro-onduleurs, Envoy, ...) en triphasé. Cela permet entre autre, de pouvoir installer plus de puissance et de pouvoir très facilement installer des panneaux par la suite.
En ce qui concerne la puissance, cela dépend bcp de ses besoins, de son "mode de consommation", de son budget, surface disponible...
La société Mk2PVRouter.com commercialise des routeurs triphasés...
Fred

**Comment by Paul-Che** [comment]
Bonsoir Fred, 
Merci beaucoup pour votre retour, c'est très gentil de votre part. Je prends note de vos conseils. J'ai également une prise intelligente sur laquelle est branchée mon BEC mécanique avec résistance électrique, j'aimerais pouvoir le piloter à distance mais à part on/off sur le passage de courant je n'ai pas la possibilité de forcer la résistance à chauffer à telle heure. 
J'ai entendu parler d'un module pour régler/forcer la puissance de chauffe d'un BEC. Est-ce que c'est pertinent pour planifier ensuite les périodes de chauffe en journée ou c'est déjà pris en compte avec la gestion du surplus du systeme Enphase ? 
ps : j'ai mis en place un système de chauffage basé sur des murs en terre chauffant, très forte inertie car environ 10cm de mur et donc j'ai besoin d'un système qui chauffe à basse température comme le plancher chauffant. De ce fait, j'aimerais en journée utiliser l'énergie PV afin de chauffer l'eau et forcer la circulation de mon chauffage.  Pas de problème si je coupe la nuit, la montée en température des murs pourra prendre quelques jours si nécessaire. 
Voila tout cela est expérimental pour moi donc l'avis d'expert m'est précieux. 
Encore merci pour vos conseils.
Paul

**Reply by FredM67** [reply, marked as answer]
Bonsoir Paul,
Hm... une prise intelligente pour alimenter un chauffe-eau, c'est moyen, très moyen... et surtout pas homologué !
"prise intelligente... pouvoir le piloter à distance mais à part on/off sur le passage de courant je n'ai pas la possibilité de forcer la résistance à chauffer à telle heure"... je ne comprends pas tout... une prise intelligente permet justement, entre autre, de programmer des périodes on/off !
"un module pour régler/forcer la puissance de chauffe", c'est-à-dire ??
Le système Enphase ne gère aucun surplus, il gère les micro-onduleurs, fournit des données de production, ...
Peut-être serait-il plus facile/simple d'en discuter par mail et/ou en direct au téléphone?
Mon adresse : frederic.metrich at live.fr

**Comment by ricardojapanese** [comment]
Bonjour Fred, j'ai récemment installé mon routeur de chez F1atb.fr, j'ai suivi votre schéma de modification du ballon d'eau chaude, ancienne version (sans le disjoncteur 2A), pour un kitable, ACI avec contacteur J/N. Cela marchait bien, jusqu'au moment où, je ne sais pas pourquoi j'ai fait ça !!! --> j'ai laissé en marche forcé (oubli) le contacteur à 1.... du coup, le ballon avait chauffé par l'excédent solaire la journée, puis par le J/N de 22h30 à 6h30... où la chauffe s'est arrêtée. Du coup évidemment, la sonde de T°C n'a pas jouée son rôle et le ballon a bien chauffé l'eau, vous pouvez imaginer. La carte ACI est toujours allumée en vert, mais le voyant orange ne s'allume jamais, alors que c'est lui qui s'allumait lorsque le contacteur J/N s'enclenchait le soir (donc quand la sonde autorisait la chauffe je pense. Du coup je pense que c'est la sonde de T°C qui a souffert, je viens d'en commander une autre, qu'en pensez-vous ? est-ce que vous pensez que c'est la carte ACI qui peut avoir été détériorée aussi ? Il semble que la sonde soit tout de même un peut cramée... La résistance est bonne car j'arrive à faire chauffer en mode forcée par le routeur pendant un certains temps.
Je vais rajouter un disjoncteur 2A pour la protection du contacteur comme la mise à jour de votre schéma.
Mais je ne comprends pas votre avertissement sur le contacteur restant collé ? Pour moi, c'est la carte ACI qui pilote le contacteur en fonction de la T°C qui est mesurée dans le ballon non ? Ce que vous dite c'est si celui-ci n'était plus piloté correctement ? ce qui finalement a été le cas pour moi car je l'avais laissé à 1 manuellement ?
En vous remerciant, bien cordialement, Richard.

**Reply by FredM67** [reply]
Salut Richard,
Ce que tu as fait en mettant le contacteur sur 1 (marche forcée) est exactement ce qui pourrait arriver si jamais le contacteur devenait défectueux avec ses contacts collés. C'est LA faille de sécurité que je mentionne.
La surchauffe a provoqué la fonte du fusible thermique, il faudra donc le changer. Théoriquement, rien d'autre n'est endommagé.
Je préconise l'utilisation d'un contacteur simple (pas de version HC), car les contacteurs simples ne peuvent pas être mis en marche forcée à la main. Par contre, la faille de sécurité existe toujours. Le risque est très faible, car ce type de contacteurs est prévu pour la puissance, mais bon, le risque n'est pas nul. Si on veut minimiser encore plus le risque, il faut mettre 2 contacteurs en série.
Mais cela ne sera jamais aussi sûr que ce qui a été utilisé sur les cartes ACI, à savoir 2 relais en série (sur la version tri) ou un relais sur neutre et un sur phase, avec un des 2 relais collé à la mise sous tension de la carte AVANT toute commutation de puissance. Cette conception permet de garantir qu'un relais sera collé SANS arc électrique (puissance non commutée), et donc on peut considérer que ses contacts seront toujours "neufs"  et donc pleinement fonctionnels.
Fred

**Comment by ricardojapanese** [comment]
Bonjour Fred,
Merci pour ta réponse et tes explications, c'est super. En effet, j'avais
mesuré la résistance entre les 2 fils bleu et j'ai trouvé une valeur très
élevée alors que entre les 2 fils jaune, une valeur proche de 9.5k à T°C
ambiante, donc je pense que en effet, le thermique s'est coupé.
C'est en cherchant que j'ai compris que le fusible thermique était intégré
à la sonde de T°C.
Je reçois la sonde aujourd'hui et j'ai commandé un disjoncteur C2 pour
compléter le schéma. Je conserve mon contacteur, mais il est dans un
boîtier avec couvercle, et il n'y a que moi qui le manipule, mais je
mettrais une note pour ne pas oublier !! ou si quelqu'un devait intervenir.
Je comprends mieux le fonctionnement maintenant.
En te remerciant encore et aussi pour tes schémas pour modifier le ballon
afin qu'il soit compatible avec un routeur, c'est vraiment top.
Verdict ce soir !
Bien cordialement,
Richard.



Le lun. 13 mai 2024 à 22:35, Frédéric Metrich ***@***.***> a
écrit :

**Reply by FredM67** [reply]
Oui, la sonde de température ainsi que le fusible thermique sont intégrés dans le même "tube".
Indépendamment de cette mésaventure, si ce n'est pas déjà fait, je te conseille vivement (depusi quelques années, c'est obligatoire) de mettre une vanne thermostatique en sortie de chauffe-eau. Cette vanne a un triple avantage :
- sécurité aux brûlures
- préservation des tuyaux, notamment s'ils sont en PE (le cuivre supportera sans problème 90°C, mais les PE, c'est pas pareil.
- économie d'énergie : au lieu d'envoyer de l'eau à 60-65°C dans les tuyaux et rajouter de l'eau froide au robinet, la vanne envoie de l'eau à 40-45°C (selon réglage), et tu rajoutes moins d'eau froide au bout.

Fred

**Comment by ricardojapanese** [comment]
Oui, il y a un groupe de sécurité thermostatique, c'est sans doute lui qui
a permis de libérer la surpression créée par la chauffe, mais je n'étais
pas là pour voir si elle fonctionnait bien, je pense que oui. (je l'ai
actionné d'ailleurs manuellement pour vérifier). Ensuite nous avons des
robinets thermostatiques partout, ce qui permet de sortir à 52°C maxi
(mesure à la cuisine).
Je te dirai si tout fonctionne après l'intervention.
Merci!
Richard.

Le mar. 14 mai 2024 à 09:51, Frédéric Metrich ***@***.***> a
écrit :

**Reply by FredM67** [reply]
Non, je ne parle pas du groupe de sécurité, qui permet d'éviter que le chauffe-eau explose.
Les robinets thermostatiques, c'est une chose, mais la dernière norme prévoit l'installation d'une vanne thermostatique en sortie de chauffe-eau.

**Comment by ricardojapanese** [comment]
Ah désolé, je comprends Frederic, je regarderai, je n'ai pas fait attention
si il y avait cette protection.
En tout cas, j'ai reçu la sonde, mise en place ce midi, nickel, le voyant
orange s'est allumé et le contacteur s'est enclenché ! ouf.
C'est top.
Merci de vos conseils et disponibilité !
Bien cordialement,
Richard.

Le mar. 14 mai 2024 à 11:02, Frédéric Metrich ***@***.***> a
écrit :

**Comment by papy-rus** [comment]
Bonjour Fred,

				J'habite en Savoie sur les hauteurs d'Ugine à 1200m d'altitude. Je viens d'installer, en début d'automne, 12 panneaux photovoltaïques, sur châssis au sol exposés plein sud, fournis par Oscaro Power,montés par mes soins.

	L'installation est doté d'un onduleur tri Huawei Sun 2000
	La Puissance de Raccordement est égale à 5.16 kVA en triphasé répartie 1.5 kVA/1.5 kVA/2.16 kVA. 
	Equipé d'un compteur Smart Power Sensor, et d'une supervision Fusion Solar.

	Après avoir pris connaissance du produit de Robin Emley,  je suis intéressé par le MK2PVrouter Triphasé.
	Afin de ne pas faire d'impair, je vous présente mon projet.
	Dans la fin des années 60, j'ai passé un bac électrotechnique,simplement pour vous dire qu'il me reste une bonne base en électricité, mais par contre je suis moyen en électronique et encore plus en informatique.

		Le tableau général d'installation triphasé  refait au printemps comporte  tous les éléments de sécurité en vigueur.
	Je bénéficie du tarif EJP d'EDF,18 KVA depuis longtemps, j'ai donc un Multi 9 qui gère le signal EJP et la coupure des gros postes de consommation tel le chauffe eau, plancher chauffant,radiateurs.a travers des contacteurs triphasés 4 pôles 3 positions, également une horloge  pour le chauffe eau
 	Mon chauffe eau est en triphasé 400v, capacité 300l, puissance 3Kw, avec un thermostat mécanique tel que sur vos photos, et une anode stéatite dont la carte est alimenté en 240v.
	Je vais le brancher en triphasé sans neutre tel votre schéma
	Le plancher chauffant a une puissance de 2750w, commandé par un module électronique ACSO.
			Mes besoins sont:

	Commander le chauffe eau pour coller au maximum à ma production.
	Pouvoir gérer mon plancher chauffant par contact oui ou non
	Avoir un contact oui ou non en attente.

Si j'ai bien compris, je n'ai besoin que de 2 triacs.
Est' il possible de ponter le comptage sur l'arrivée du compteur Smart Power sensor?
Pour l'anode du chauffe eau je pense tirer une ligne 240v indépendante.
Je n'ai pas compris si une communication WiFi est possible.
La sonde de température est elle obligatoire et quelle est sont utilité réelle.
Pour la partie programmation j'ai un ami voisin qui pourrait m'aider.

Voilà pourquoi j'envisage l'achat du Kit expert triphasé PV Router sans RF, et je voudrais savoir ce que je dois préciser dans ma commande.
C'est peut être beaucoup vous demander, mais mes modestes connaissances ont besoin d'un sérieux coup de main.
Si toutefois c'est trop vous demander par rapport à votre temps, je comprendrais.
Bien cordialement

							Pierre Sigaud

**Comment by FredM67** [comment]
Salut Pierre,

Je pense qu'on peut se tutoyer, c'est l'habitude sur le net 😉

Exact, chauffe-eau triphasé sans neutre, donc 2 triacs uniquement.
"Pont le comptage sur le Smart Power sensor", c'est-à-dire ?

En l'état actuel, les mk2pvrouter n'ont aucune interface Wifi, ils fonctionnent en totale autonomie.
Il est cependant possible de rajouter tel ou tel module, à voir à quoi cela pourrait servir.
Ce que je rajoute le plus souvent est une carte relais wifi qui permet de piloter à distance et/ou via scenarii et/ou domotique la mise en route de marche forcée par exemple.
La sonde de température est optionnelle. Elle permet d'affiner certaines fonctions comme une marche forcée asservie à la température.

Le prog existe déjà, il faut simplement l'adapter en fonction de son cas personnel. J'ai écrit de la doc pour cela, et de toute façon, je suis toujours dispo pour tout renseignement et/ou aide.

Donc pour la commande, il faudrait au mini:
- mk2pvrouter triphasé sans RF
- 2 sorties triac
- 2 sorties relais

Fred

PS: tu peux m'écrire directement sur mon mail (mon adresse figure en entête dans la plupart des fichiers sur github)


Envoyé à partir d’Outlook<http://aka.ms/weboutlook>

________________________________
De : papy-rus ***@***.***>
Envoyé : mercredi 4 décembre 2024 15:11
À : FredM67/PVRouter-3-phase ***@***.***>
Cc : Frédéric Metrich ***@***.***>; Comment ***@***.***>
Objet : Re: [FredM67/PVRouter-3-phase] Demande d'information (Discussion #9)


                        Bonjour Fred,

                        J'habite en Savoie sur les hauteurs d'Ugine à 1200m d'altitude. Je viens d'installer, en début d'automne, 12 panneaux photovoltaïques, sur châssis au sol exposés plein sud, fournis par Oscaro Power,montés par mes soins.

L'installation est doté d'un onduleur tri Huawei Sun 2000
La Puissance de Raccordement est égale à 5.16 kVA en triphasé répartie 1.5 kVA/1.5 kVA/2.16 kVA.
Equipé d'un compteur Smart Power Sensor, et d'une supervision Fusion Solar.

Après avoir pris connaissance du produit de Robin Emley,  je suis intéressé par le MK2PVrouter Triphasé.
Afin de ne pas faire d'impair, je vous présente mon projet.
Dans la fin des années 60, j'ai passé un bac électrotechnique,simplement pour vous dire qu'il me reste une bonne base en électricité, mais par contre je suis moyen en électronique et encore plus en informatique.

        Le tableau général d'installation triphasé  refait au printemps comporte  tous les éléments de sécurité en vigueur.
Je bénéficie du tarif EJP d'EDF,18 KVA depuis longtemps, j'ai donc un Multi 9 qui gère le signal EJP et la coupure des gros postes de consommation tel le chauffe eau, plancher chauffant,radiateurs.a travers des contacteurs triphasés 4 pôles 3 positions, également une horloge  pour le chauffe eau
Mon chauffe eau est en triphasé 400v, capacité 300l, puissance 3Kw, avec un thermostat mécanique tel que sur vos photos, et une anode stéatite dont la carte est alimenté en 240v.
Je vais le brancher en triphasé sans neutre tel votre schéma
Le plancher chauffant a une puissance de 2750w, commandé par un module électronique ACSO.
                Mes besoins sont:

Commander le chauffe eau pour coller au maximum à ma production.
Pouvoir gérer mon plancher chauffant par contact oui ou non
Avoir un contact oui ou non en attente.


Si j'ai bien compris, je n'ai besoin que de 2 triacs.
Est' il possible de ponter le comptage sur l'arrivée du compteur Smart Power sensor?
Pour l'anode du chauffe eau je pense tirer une ligne 240v indépendante.
Je n'ai pas compris si une communication WiFi est possible.
La sonde de température est elle obligatoire et quelle est sont utilité réelle.
Pour la partie programmation j'ai un ami voisin qui pourrait m'aider.

Voilà pourquoi j'envisage l'achat du Kit expert triphasé PV Router sans RF, et je voudrais savoir ce que je dois préciser dans ma commande.
C'est peut être beaucoup vous demander, mais mes modestes connaissances ont besoin d'un sérieux coup de main.
Si toutefois c'est trop vous demander par rapport à votre temps, je comprendrais.
Bien cordialement

                                                Pierre Sigaud

**Comment by papy-rus** [comment]
Merci Fred,
Nouveau sur ces communications pas de problème pour le tutoiement.
Le smart power sensor est un compteur communiquant avec l'onduleur,  il possède déjà 3 tores de comptage. Je pensais simplement ponter ces tores  sur les bores d'entrée au lieu d'en rajouter, mais ce n'est peut être pas conseillé.
La carte Wifi peut elle être fourni sur le site "mk2"?
Dois je préciser la fourniture de la sonde t°?
J'aurais surent besoin de tes conseils plus tard.
Merci encore
Pierre

**Comment by FredM67** [comment]
Re,

Non, on ne peut pour ainsi dire jamais utiliser des pinces ampèremétriques sur plusieurs appareils en même temps.
Elles fonctionnent en analogique, il y a donc tout un tas de paramètres qui entrent en jeux (impédance notamment).

Je pense que Quentin peut te fournir la carte Wifi... en général, je les prends sur Amazon (en version Tuya).
Idem pour la sonde, il faut demander à Quentin.

No soucy pour les conseils avant/après 😉 même bien après 😄

Fred


Envoyé à partir d’Outlook<http://aka.ms/weboutlook>

________________________________
De : papy-rus ***@***.***>
Envoyé : mercredi 4 décembre 2024 16:34
À : FredM67/PVRouter-3-phase ***@***.***>
Cc : Frédéric Metrich ***@***.***>; Comment ***@***.***>
Objet : Re: [FredM67/PVRouter-3-phase] Demande d'information (Discussion #9)


Merci Fred,
Nouveau sur ces communications pas de problème pour le tutoiement.
Le smart power sensor est un compteur communiquant avec l'onduleur, il possède déjà 3 tores de comptage. Je pensais simplement ponter ces tores sur les bores d'entrée au lieu d'en rajouter, mais ce n'est peut être pas conseillé.
La carte Wifi peut elle être fourni sur le site "mk2"?
Dois je préciser la fourniture de la sonde t°?
J'aurais surent besoin de tes conseils plus tard.
Merci encore
Pierre

**Comment by papy-rus** [comment]
Bonsoir Fred,
Oui, je suis toujours là.
J'ai échangé avec Quentin, je te transfère notre prose, tu me feras tes remarques.
Merci 
Pierre

**Comment by papy-rus** [comment]
Bonjour Fred,
Trop d'information tue l'information.
Je t'ai noyé d'échanges, et je m'en excuse.
Pour être plus concis, en pièce jointe , les 2 dernières réponses d'Energie Router, et un schéma qu'is ont complétés.
Il manquerait la sonde température du chauffe eau.
L'ensemble parait cohérent.
Le routeur sera a coté de mon tableau
Pas de souci pour le positionnement des sorties triacs du chauffe eau, la sonde t° a tirer sur 3ml.
Il me faudra déporté le triac plancher chauffant à 5ml du MK2, et la sonde t° à 6,5 ml.
Ce qui pose question, c'est le rappel qu'ils font sur ce projet sur mesure: "pas de notice de montage pour les thermostats embarqués et les sorties relais.
Y  a t'il une solution, ou dois je passer par toi.
J'attends ta réponse, car j'ai hâte d'avancer sur ce projet.
Merci
Pierre



Pierre Sigaud
***@***.***

**Reply by FredM67** [reply]
Non, c'est pas cela du tout.. j'ai eu un weekend méga chargé !! Quentin m'a transmis les mails... je te réponds demain....

**Comment by papy-rus** [comment]
Merci Fred

**Comment by papy-rus** [comment]
Bonjour Fred, 
je te joins 2 pdf copie de mail avec Quentin , 1 que tu as déjà, ainsi que le schéma que tu as du voir aussi.
Comme je l'ai déjà dit, j'ai d'assez bonnes connaissances en électricité, moins bonnes en électronique;
Si j'ai un plan bien détaillé pas à pas, je pense pouvoir monter correctement  la carte avec votres aides.
Et mon collègue est bon en programmation, il participe régulierement à des programmes open sources dans les domaines géographiques de haut niveau. C'est d'ailleurs lui qui échangera avec toi ou Quentin quand j'en serais là.
Il me faudrait donc cette notice de montage pour thermostats embarqués et sortie relais.
Oui, pour la sortie relais, j'ai envie d'en mettre une, pour l'avoir au cas ou, mais je n'ai pas d'idée précise, peut être commande con tacteur chauffage  gîte.
Si je peux avoir tout cela je vais me lancer.
Donc, je confirme qu'au niveau matériel, il faut t rajouter la sonde chauffe eau  et le 3° trials déportés
Es t'il possible que tu vois avec Quentin pour lister le matériel nécessaire, et je finaliserai ma commande.
Merci 
Pierre

Copie de mon dernier mail
Pour être plus concis, en pièce jointe , les 2 dernières réponses d'Energie Router, et un schéma qu'is ont complétés.
Il manquerait la sonde température du chauffe eau.
L'ensemble parait cohérent.
Le routeur sera a coté de mon tableau
Pas de souci pour le positionnement des sorties triacs du chauffe eau, la sonde t° a tirer sur 3ml.
Il me faudra déporté le triac plancher chauffant à 5ml du MK2, et la sonde t° à 6,5 ml.
Ce qui pose question, c'est le rappel qu'ils font sur ce projet sur mesure: "pas de notice de montage pour les thermostats embarqués et les sorties relais.

**Reply by FredM67** [reply]
Stp, réponds au mail que je t'ai envoyé ce matin... les "Discussions" sur Github ne sont pas faites pour cela... je ne peux pas m'en sortir avec les réponses, les pièces jointes (que je ne reçois pas d'ailleurs), ...
